### PR TITLE
#32 Refractored directory structure

### PR DESCRIPTION
**Pull request by shriaas2898**
* Changed `DocumentRoot` to `public-html`.
* Moved `asset` to `public-html`
* Changed path of `autoload.php` in `SessionCookie` and `dbConn`

To use `autoload` add this line:
```
require_once($_SERVER["DOCUMENT_ROOT"]."/../vendor/autoload.php");
```
and 
```
use Namspace\Classname
```

Replace `Namespace`  and `Classname` according you your requirement.

 @AJV009 @anjali-rathod @pragzii3896

### CODE REVIEW

**Comment by shriaas2898** [comment]
@JayKandari, @guptahemant   please review it.

**Review by guptahemant — CHANGES_REQUESTED** [review]
@shriaas2898 can you please update in the readme as well, that virtual host for this site should point to public_html folder

File: resources/src/config.php [hunk 1]
```diff
@@ -5,7 +5,7 @@ class ProjectConfig
     public $config;
     public function __construct()
     {
-        $creds = json_decode(file_get_contents($_SERVER['DOCUMENT_ROOT'].'/dbcreds.json'), true);
+        $creds = json_decode(file_get_contents($_SERVER["DOCUMENT_ROOT"]."/../resources/dbcreds.json"), true);
```
Comment (AJV009) [inline resources/src/config.php:8]: And also that you have changed the dbcreds from root to resources

File: resources/src/config.php [hunk 2]
```diff
@@ -21,7 +21,7 @@ public function __construct()
                 ),
                 // Paths of all the required folders
                 'paths' => array(
-                    "resources" => "/resources",
+                    "resources" => "/../resources",
                     "images" =>  "/asset/images",
                     "css" =>  "/asset/css",
                     "js" => "/asset/js"
```
Comment (AJV009) [inline resources/src/config.php:27]: So is it now pointing to folder inside public-html?

File: resources/src/config.php (hunk 1 above)
Comment (shriaas2898) [reply resources/src/config.php:8]: I have not moved `dbcreds`, I have just changed its path

File: resources/src/config.php (hunk 2 above)
Comment (shriaas2898) [reply resources/src/config.php:27]: resources is **outside** public-html, hence `/../`. And  `asset` is inside public-html

**Comment by shriaas2898** [comment]
> @shriaas2898 can you please update in the readme as well, that virtual host for this site should point to public_html folder

I have added the requested changes

**Comment by AJV009** [comment]
@guptahemant  can we get this merged asap
as a lot of paths have changed we all are depended on this one

**Review by guptahemant — APPROVED** [review]
Looks good, though we might want to put dbcreds.json in the project root, instead of inside the resources folder.

**Comment by AJV009** [comment]
> Looks good, though we might want to put dbcreds.json in the project root, instead of inside the resources folder.

Yeah I had kept it in the project root BUT em leave it maybe next time next project we will have dbcreads placed in root.
because now some cant wait for that small change of path in config for dbcreds

**Pull request merged**